<font size=4 face='楷体'>

在 python3.7 中 `Crypto`和`pycrypto`包都很问题，不能正常使用  
采用新的`pycryptodome`即可

[Link](https://blog.csdn.net/zoulonglong/article/details/79552813)